### ISSUE TITLE
Feature suggestion - User groups admin

### ISSUE THREAD

**Issue by ki6uve** [brian7704/OpenTAKServer#36]
Suggestion for adding the ability to admin users by group(s).  Users can be added to one or more groups.  Groups can be enabled/disabled en masse by OTS administrator.

**Comment by sgofferj** [comment]
Yeah, that definitely makes sense. Especially if those groups are taken into account in CoT routing. Routing based on team colors is fine but IMHO there should also be a way of controlling the routing by the admin that the users can't influence.

**Comment by nerdalertdk** [comment]
Nice with more group fans :D
Already created a group request, but was only for the OG tak group support 

https://github.com/brian7704/OpenTAKServer/issues/31
